docs/testing/ie-11-testing.md

```diff
@@ -28,7 +28,7 @@ If you plan to market your add-in through AppSource or you plan to support older
 > Also, Internet Explorer 11 does not support some HTML5 features such as media, recording, and location.
 
 > [!NOTE]
-> To test your add-in on the Internet Explorer 11 browser, open Office on the web in Internet Explorer and [sideload the add-in](create-a-network-shared-folder-catalog-for-task-pane-and-content-add-ins.md).
+> Office on the web cannot be opened in Internet Explorer 11, so you cannot (and do not need to) test your add-in on Office on the web with Internet Explorer.
```

Comment:
Would setting the compatibility mode or user agent to Internet Explorer in a supported browser's dev tools be useful/helpful, particularly for testing add-ins that support Office versions running IE webview?

Reply:
I don't know. But that would be out-of-scope for this PR anyway. This is only about IE not supporting Office on the web, not about general IE testing advice.